src/dns_internal.cc

```diff
@@ -53,9 +54,6 @@
 #define REG_TCPIP_PARA "SYSTEM\\CurrentControlSet\\Services\\Tcpip\\Parameters"
 #define REG_VXD_MSTCP "SYSTEM\\CurrentControlSet\\Services\\VxD\\MSTCP"
 #endif
-#ifndef _PATH_RESCONF
```

Comment:
This macro is supposed to be provided by resolv.h in all the currently supported glibc. so it should no longer be needed. If we have issues it can be added back in ResolvConf.h or as a configure.ac definition.